soundbay/data.py

```diff
@@ -358,7 +360,7 @@ def _create_start_times(self):
             audio_dict contains references to audio paths given name from metadata
         """
         audio_len = sf.info(self.file_path).duration
-        self._start_times = np.arange(0, audio_len//self.seq_length * self.seq_length, self.seq_length).astype(int)
+        self._start_times = np.arange(0, audio_len//self.seq_length * self.seq_length, self.seq_length)
```

Comment:
doesn't it make sense to change this to / rather than // ?

Reply:
I think it would make the array contain a start time of a segment that's distance from the end of the wav is shorter than seq_length.
e.g.:
len=audio_len = 10.1 sec, and seq=seq_length=0.2 would give us:
np.arange(0,len//seq*seq,seq) = 0, 0.2, 0.4, ..., 9.6, 9.8
np.arange(0,len/seq*seq,seq) = 0, 0.2, 0.4, ..., 9.6, 9.8, 10
so the sequence starting at 10 with seq length 0.2 would exceed the wav length

Reply:
oh understood